terrautils/metadata.py

```diff
@@ -119,6 +119,8 @@ def pipeline_get_metadata(metadata):
             1) "content" -> "pipeline"
             2) "pipeline"
     """
+    from terrautils.secure import decrypt_pipeline_string
```

Comment:
any reason not to have this at the top with the rest of the imports?

Reply:
@robkooper I put it here so that there's one less script to load if it's not needed; it's pretty specialized at this time. I can put it up top, or do something else if you want it moved.

Reply:
I tend to like imports at the top. That being said, I think this secure thing might be something we propegate through more of the code.

Reply:
Made the change